### ISSUE TITLE
Search is not returning any results

### ISSUE THREAD

**Issue by johndierks** [facundoolano/google-play-scraper#305]
It looks like the DOM structure of the search results page on the play store has changed and is not getting parsed correctly any more.

Unfortunately it seems they have done away with human readable selectors and are now using generated selectors that I presume will change when they rebuild/redeploy the play store.

```
var gplay = require('google-play-scraper');

gplay.search({
    term: "panda",
    num: 2
}).then(console.log, console.log);
```

returns no results.

```
[]
```

**Comment by johndierks** [comment]
For what it's worth, this got the search method partly working for me. Many of the tests are still failing though.

https://github.com/johndierks/google-play-scraper/commit/82db439961d3f8181b6c7c88349e96d77085eaac

**Comment by hunnapikes** [comment]
Work, but with limit 50.
https://github.com/hunnapikes/google-play-scraper/commit/2aab61c7c27d829d491fe71185d50e89f7b17e3d

**Comment by facundoolano** [comment]
Regarding this, considering it looks like google is moving towards the data embedded in scripts approach, I want to get #303 approved and merged first, since it's the one that's more far along and where we've been discussing the changes. 

Once that is merged, the fix for `search` (and `developer` maybe?) should be based on those new helpers.

**Comment by hunnapikes** [comment]
With fixed limit.

https://github.com/hunnapikes/google-play-scraper/commit/d036cd954f3df6a616e690ffc808c6ff69c31484

**Comment by KaKi87** [comment]
Not working for me whatever limit number I use

**Comment by Joepruner** [comment]
Hunnapikes fix worked for me, but only after realizing there was another branch name "fix-search". Doh

**Comment by fenixdp** [comment]
fix-search now get only 50 results. Any ideas how to increase up to 250 as early?

**Comment by hexdump2002** [comment]
Where do you provide that limit param?

**Comment by KaKi87** [comment]
@hunnapikes Why not publishing to npm instead ?

**Comment by KaKi87** [comment]
Now I've got this error with the last published version at npmjs.com and a 50 number limit :
```
{ Error: Error requesting Google Play:undefined
    at /opt/KPlaySearch/node_modules/google-play-scraper/lib/utils/request.js:42:19
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7) status: 429 }
```

**Comment by bumsun** [comment]
I am also waiting for fix

**Comment by blank003** [comment]
How do you set the limit.Can anyone explain.?

**Comment by facundoolano** [comment]
This should be fixed by #303

**Comment by KaKi87** [comment]
Why `minInstalls` is now missing from `search` promise response with `fullDetail` as `true` ?

**Comment by Alphabommel** [comment]
It seems that `fullDetail` is not working with the current search method imlpementation.

**Comment by facundoolano** [comment]
can you open a separate issue with that? thanks!
